package.json

```diff
@@ -26,23 +26,20 @@
     "build:generators": "gulp buildGenerators",
     "build:langfiles": "gulp buildLangfiles",
     "build:uncompressed": "gulp buildUncompressed",
-    "bump": "npm version 3.$(date +'%Y%m%d').0",
+    "bump": "npm --no-git-tag-version version 3.$(date +'%Y%m%d').0",
     "license": "gulp checkLicenses",
     "lint": "eslint .",
     "package": "gulp package",
-    "postversion": "gulp postversion",
     "prepare": "npm run package",
-    "preversion": "gulp preversion",
-    "prerelease": "gulp gitRecompile",
```

Comment:
Are there any references to postversion, preversion, version, or prelease that you need to update?

Reply:
I don't think so. These are automagically called when someone runs npm run version

Reply:
Even prerelease?

Reply:
I think so. Running npm prerelease was our first step in our old release instructions, so I don't think it got used internally.